draft/spec/validation-codes.md

```diff
@@ -113,6 +113,8 @@
 | E101 | 'Within an inventory, content paths must be unique and non-conflicting, so the content path for a file cannot appear as the initial part of another content path.' | https://ocfl.io/draft/spec/#E101
 | E102 | 'An inventory file must not contain keys that are not specified.' | https://ocfl.io/draft/spec/#E102
 | E103 | 'Each version directory within an OCFL Object MUST conform to either the same or a later OCFL specification version as the preceding version directory.' | https://ocfl.io/draft/spec/#E103
+| E106 | 'The value of the manifest key MUST be a JSON object.' | https://ocfl.io/draft/spec/#E106
+| E107 | 'The value of the manifest key must be a JSON object, all keys of which MUST correspond to digest value keys found in one or more state JSON objects of the current and/or previous version blocks of the OCFL Object.' | https://ocfl.io/draft/spec/#E107
```

Comment:
A minor point, but the text of the error code is using the previous language:
> all keys of which MUST correspond..

instead of:
> and each key MUST correspond..

Aligning the language of the error code to the spec seems more even more clear.

Reply:
Well spotted @awoods ! I have updated the validation codes text to match the improved spec language